cmd/root/root.go

```diff
@@ -82,21 +83,17 @@ func defaultPidFilePath() string {
 	// Otherwise, use /run
 	path := "/run/pget.pid"
 	if runtime.GOOS == "darwin" {
-		path = os.Getenv("HOME") + "/.pget.pid"
+		path = filepath.Join(os.Getenv("HOME"), ".pget.pid")
```

Comment:
os.UserHomeDir would be slightly nicer. Or XDG_RUNTIME_DIR if it’s set (but not sure if that’s a macOS thing)